Libraries/LibELF/ELFLoader.cpp

```diff
@@ -18,6 +18,27 @@ ELFLoader::~ELFLoader()
 {
 }
 
+Optional<String> ELFLoader::interpreter()
+{
+    String interpreter_path;
+    m_image.for_each_program_header([&](const ELFImage::ProgramHeader& program_header) {
+        if (program_header.type() == PT_INTERP) {
+            if (!m_image.is_within_image(program_header.raw_data(), program_header.size_in_image())) {
+                dbg() << "Shenanigans! ELF PT_INTERP header sneaks outside of executable.";
+            }
+            else {
```

Comment:
This is not how clang-format normally indents this:
```suggestion
            } else {
```

Reply:
Whoops you're right. ran clang-format for my latest push